docs/user/guides/custom-domains.rst

```diff
@@ -0,0 +1,119 @@
+How To Manage Custom Domains
+============================
+
+This guide describes how to host your documentation using your own domain name, such as ``docs.example.com``.
+
+.. contents:: Contents
+    :local:
+
+Adding a custom domain
+----------------------
+
+To setup your :doc:`custom domain </custom-domains>`, follow these steps:
+
+#. Go the :guilabel:`Admin` tab of your project.
+#. Click on :guilabel:`Domains`.
+#. Enter the domain where you want to serve the documentation from (e.g. ``docs.example.com``).
+#. Mark the :guilabel:`Canonical` option if you want use this domain
+   as your :doc:`canonical domain </canonical-urls>`.
+#. Click on :guilabel:`Add`.
+#. At the top of the next page you'll find the value of the DNS record that you need to point your domain to.
+   For |org_brand| this is ``readthedocs.io``, and for :doc:`/commercial/index`
+   the record is in the form of ``<hash>.domains.readthedocs.com``.
+
+   .. note::
+
+      For a subdomain like ``docs.example.com`` add a CNAME record,
+      and for a root domain like ``example.com`` use an ANAME or ALIAS record.
+
+By default, we provide a validated SSL certificate for the domain,
```

Comment:
What does the "by default" part mean? Are there other options? CC: @ericholscher

Reply:
What I mean is that maybe we should a) remove "By default" or b) explain if there are other options (perhaps in a separate tip box)

Reply:
Yea, I think we just remove the `by default`.

Reply:
Rm'ed!